pkgs/servers/openvscode-server/default.nix

```diff
@@ -44,74 +39,69 @@ let
     }
     .${system} or (throw "Unsupported system ${system}");
 
-  esbuild' = esbuild.override {
-    buildGoModule =
-      args:
-      buildGoModule (
-        args
-        // rec {
-          version = "0.17.14";
-          src = fetchFromGitHub {
-            owner = "evanw";
-            repo = "esbuild";
-            rev = "v${version}";
-            hash = "sha256-4TC1d5FOZHUMuEMTcTOBLZZM+sFUswhyblI5HVWyvPA=";
-          };
-          vendorHash = "sha256-+BfxCyg0KkDQpHt/wycy/8CTG6YBA/VJvJFhhzUnSiQ=";
-        }
-      );
-  };
-
-  # replaces esbuild's download script with a binary from nixpkgs
-  patchEsbuild = path: version: ''
-    mkdir -p ${path}/node_modules/esbuild/bin
-    jq "del(.scripts.postinstall)" ${path}/node_modules/esbuild/package.json | sponge ${path}/node_modules/esbuild/package.json
-    sed -i 's/${version}/${esbuild'.version}/g' ${path}/node_modules/esbuild/lib/main.js
-    ln -s -f ${esbuild'}/bin/esbuild ${path}/node_modules/esbuild/bin/esbuild
-  '';
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "openvscode-server";
-  version = "1.88.1";
+
+  inherit nodejs version;
 
   src = fetchFromGitHub {
     owner = "gitpod-io";
     repo = "openvscode-server";
-    rev = "openvscode-server-v${finalAttrs.version}";
-    hash = "sha256-Yc16L13Z8AmsGoSFbvy+4+KBdHxvqLMwZLeU2/dAQVU=";
+    rev = "openvscode-server-v${version}";
+    inherit hash;
```

Comment:
For automatic updates by `nix-udpate`, I'm pretty sure that putting the hash and version like in every package.

Reply:
Is that already enough to make nix-update work? I didn't see any updates before.

In any case, I reverted the position of version and hashes, since I had only moved it up, because I couldn't use `finalAttrs` with `buildNpmPackage`, which I ended up not using after all.

Reply:
> Is that already enough to make nix-update work? I didn't see any updates before.

`nix-update` is a tool meant for maintainers, whereas the [`nixpkgs-update`](https://github.com/nix-community/nixpkgs-update) bot is also working well for simple enough packages where increasing the version and replacing the hash is all needed for updating a package. I don't know enough about it, but the mechanism used by these tools to replace the old version and hash attributes is what requires their location in the expression to be in a typical location.